# The optimal motion plan represented in cartesian space

 RobotStateCartesian[] states    # trajectory of states
 float64[]                  t    # time at corresponding state
